telegram/ext/filters.py

```diff
@@ -372,1895 +363,1807 @@ def name(self, name: str) -> NoReturn:
 
 
 class _DiceEmoji(MessageFilter):
-    __slots__ = ('emoji',)
+    __slots__ = ('emoji', 'values')
 
-    def __init__(self, emoji: str = None, name: str = None):
-        self.name = f'Filters.dice.{name}' if name else 'Filters.dice'
+    def __init__(self, values: SLT[int] = None, emoji: str = None):
+        name = f"filters.DICE.{getattr(emoji, 'name', '')}" if emoji else 'filters.DICE'
```

Comment:
Yes, first I just did `emoji.name`, then one of the tests failed so I figured this is safer and better. It also got pycharm to shut up about string has no attribute `name`.

My main struggle was to type hint a string which is from a `Enum`, but it can't find any.

Reply:
ah, we're passing enums here :D then probably just annotate as `emoji: DiceEmoji` - the class is private anyway, right?